js/sound.js

```diff
@@ -1,5 +1,12 @@
 /*jshint esversion: 6 */
 const backgroundAudio = [
+    {
+        filename: "audio/0x48piraj/composed/0x48piraj_Rusted_Heart.mp3",
```

Comment:
I cannot find this in the diff

Reply:
I have added this in a previous Pull Request :+1: